packages/components/src/internal/components/assay/__snapshots__/RunDataPanel.spec.tsx.snap

```diff
@@ -302,7 +302,7 @@ exports[`<RunDataPanel/> custom display props 1`] = `
                         }
                         title="When a matching specimen exists in a study, can be used to identify subject and timepoint for assay. Alternately, supply ParticipantID and either VisitID or Date."
                       >
-                        Specimen ID
+                        SampleID
```

Comment:
I assume this is just a change from the data that the jest test is now using. Just want to make sure this isn't an indicator of some missing call to go through column metadata or something like that.

Reply:
That is my understanding